src/canvas-extractor/canvas_extractor/csv_generation/write.py

```diff
@@ -0,0 +1,91 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
If this is identical to the GC version, maybe put a big comment to that effect so we know to extract to a shared library.  See api/resource_sync.py for an example.

Reply:
✔️